api/desecapi/tests/test_rrsets.py

```diff
@@ -199,16 +199,16 @@ def test_create_other_rr_sets(self):
 
     def test_create_my_rr_sets_too_long_content(self):
         def _create_data(length):
-            content_string = 'A' * (length - 2)  # we have two quotes
-            return {'records': [f'"{content_string}"'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{length}'}
+            content_string = 'A' * length
+            return {'records': [f'{content_string}'], 'ttl': 3600, 'type': 'TXT', 'subname': f'name{length}'}
```

Comment:
Why remove the quotes? They are mandatory in `TXT`

Reply:
While not strictly necessary, we agreed offline to deviate from the standard and make them mandatory